lib/Client.js

```diff
@@ -304,4 +305,4 @@ function queryStringForLogging(query) {
Z2lmeSh0aGlzLl9zZWNyZXQpfWApXG4gICAgICBpZiAoZGF0YSlcbiAgICAgICAgdGhpcy5fbG9nKHRydWUsIGBSZXF1ZXN0IEpTT046ICR7dG9KU09OKGRhdGEsIHRydWUpfWApXG5cbiAgICAgIGNvbnN0IGhlYWRlcnNfanNvbiA9IHRvSlNPTihyZXNwb25zZS5oZWFkZXJzLCB0cnVlKVxuICAgICAgY29uc3QgcmVzcG9uc2Vfb2JqZWN0ID0gcGFyc2VKU09OKGJvZHkpXG4gICAgICBjb25zdCByZXNwb25zZV9qc29uID0gdG9KU09OKHJlc3BvbnNlX29iamVjdCwgdHJ1ZSlcbiAgICAgIHRoaXMuX2xvZyh0cnVlLCBgUmVzcG9uc2UgaGVhZGVyczogJHtoZWFkZXJzX2pzb259YClcbiAgICAgIHRoaXMuX2xvZyh0cnVlLCBgUmVzcG9uc2UgSlNPTjogJHtyZXNwb25zZV9qc29ufWApXG4gICAgICBjb25zdFxuICAgICAgICBzdGF0dXNDb2RlID0gcmVzcG9uc2Uuc3RhdHVzQ29kZSxcbiAgICAgICAgYXBpVGltZSA9IHJlc3BvbnNlLmhlYWRlcnNbJ3gtaHR0cC1yZXF1ZXN0LXByb2Nlc3NpbmctdGltZSddLFxuICAgICAgICBsYXRlbmN5ID0gTWF0aC5mbG9vcihyZWFsX3RpbWUpXG4gICAgICB0aGlzLl9sb2codHJ1ZSxcbiAgICAgICAgYFJlc3BvbnNlICgke3N0YXR1c0NvZGV9KTogQVBJIHByb2Nlc3NpbmcgJHthcGlUaW1lfW1zLCBuZXR3b3JrIGxhdGVuY3kgJHtsYXRlbmN5fW1zYClcbiAgICAgIHJldHVybiBoYW5kbGVSZXNwb25zZShyZXNwb25zZSwgcmVzcG9uc2Vfb2JqZWN0KVxuICAgIH1cbiAgfVxuXG4gIF9leGVjdXRlX3dpdGhvdXRfbG9nZ2luZyhhY3Rpb24sIHBhdGgsIGRhdGEsIHF1ZXJ5KSB7XG4gICAgcmV0dXJuIG5ldyBQcm9taXNlKChyZXNvbHZlLCByZWplY3QpID0+IHtcbiAgICAgIC8vIHJlcXVlc3QgaGFzIGEgYnVnIHdoZW4gdHJ5aW5nIHRvIHJlcXVlc3QgZW1wdHkgcGF0aC5cbiAgICAgIGlmIChwYXRoID09PSAnJylcbiAgICAgICAgcGF0aCA9ICcvJ1xuXG4gICAgICBjb25zdCBvcHRzID0ge1xuICAgICAgICBtZXRob2Q6IGFjdGlvbixcbiAgICAgICAgYmFzZVVybDogdGhpcy5fYmFzZVVybCxcbiAgICAgICAgdXJsOiBwYXRoLFxuICAgICAgICBhdXRoOiB0aGlzLl9zZWNyZXQsXG4gICAgICAgIHFzOiBxdWVyeSxcbiAgICAgICAgYm9keTogZGF0YSA9PT0gbnVsbCA/IG51bGwgOiB0b0pTT04oZGF0YSksXG4gICAgICAgIHRpbWVvdXQ6IHRoaXMuX3RpbWVvdXRcbiAgICAgIH1cblxuICAgICAgcmVxdWVzdChvcHRzLCAoZXJyLCByZXNwb25zZSwgYm9keSkgPT4ge1xuICAgICAgICBpZiAoZXJyKVxuICAgICAgICAgIHJlamVjdChlcnIpXG4gICAgICAgIGVsc2VcbiAgICAgICAgICByZXNvbHZlKHtyZXNwb25zZSwgYm9keX0pXG4gICAgICB9KVxuICAgIH0pXG4gIH1cbn1cblxuZnVuY3Rpb24gaGFuZGxlUmVzcG9uc2UocmVzcG9uc2UsIHJlc3BvbnNlX29iamVjdCkge1xuICBjb25zdCBjb2RlID0gcmVzcG9uc2Uuc3RhdHVzQ29kZVxuICBpZiAoMjAwIDw9IGNvZGUgJiYgY29kZSA8PSAyOTkpXG4gICAgcmV0dXJuIHJlc3BvbnNlX29iamVjdC5yZXNvdXJjZVxuICBlbHNlXG4gICAgc3dpdGNoIChjb2RlKSB7XG4gICAgICBjYXNlIDQwMDpcbiAgICAgICAgdGhyb3cgbmV3IEJhZFJlcXVlc3QocmVzcG9uc2Vfb2JqZWN0KVxuICAgICAgY2FzZSA0MDE6XG4gICAgICAgIHRocm93IG5ldyBVbmF1dGhvcml6ZWQocmVzcG9uc2Vfb2JqZWN0KVxuICAgICAgY2FzZSA0MDM6XG4gICAgICAgIHRocm93IG5ldyBQZXJtaXNzaW9uRGVuaWVkKHJlc3BvbnNlX29iamVjdClcbiAgICAgIGNhc2UgNDA0OlxuICAgICAgICB0aHJvdyBuZXcgTm90Rm91bmQocmVzcG9uc2Vfb2JqZWN0KVxuICAgICAgY2FzZSA0MDU6XG4gICAgICAgIHRocm93IG5ldyBNZXRob2ROb3RBbGxvd2VkKHJlc3BvbnNlX29iamVjdClcbiAgICAgIGNhc2UgNTAwOlxuICAgICAgICB0aHJvdyBuZXcgSW50ZXJuYWxFcnJvcihyZXNwb25zZV9vYmplY3QpXG4gICAgICBjYXNlIDUwMzpcbiAgICAgICAgdGhyb3cgbmV3IFVuYXZhaWxhYmxlRXJyb3IocmVzcG9uc2Vfb2JqZWN0KVxuICAgICAgZGVmYXVsdDpcbiAgICAgICAgdGhyb3cgbmV3IEZhdW5hSFRUUEVycm9yKHJlc3BvbnNlX29iamVjdClcbiAgICB9XG59XG5cbmZ1bmN0aW9uIHF1ZXJ5U3RyaW5nRm9yTG9nZ2luZyhxdWVyeSkge1xuICByZXR1cm4gcXVlcnkgPyBgPyR7T2JqZWN0LmtleXMocXVlcnkpLm1hcChrZXkgPT4gYCR7a2V5fT0ke3F1ZXJ5W2tleV19YCkuam9pbignJicpfWAgOiAnJ1xufVxuIl19
```

Comment:
If people don't want to use the source map then this is just a comment.
It is annoying how changes to code end up changing the diffs like this. I made an issue: #40.